tests/docs/running-e2e-test.md

```diff
@@ -29,6 +29,9 @@ E2E tests are designed for verifying the functional correctness by replicating e
     # Do not set DAPR_TEST_ENV if you do not use minikube
     export DAPR_TEST_ENV=minikube
 
+    # Set kubernetes master ip
```

Comment:
This step is only necessary if you are testing via minikube. Perhaps this comment should reflect that.